src/main/java/es/in2/desmos/domain/services/api/impl/DomeParticipantServiceImpl.java

```diff
@@ -14,13 +13,16 @@
 @RequiredArgsConstructor
 public class DomeParticipantServiceImpl implements DomeParticipantService {
 
-    private final DomeParticipantRepository domeParticipantRepository;
+    private final TrustedAccessNodesListRepository trustedAccessNodesListRepository;
 
     @Override
-    public Mono<DomeParticipant> validateDomeParticipant(String processId, String ethereumAddress) {
-        log.info("ProcessID: {} - Validating Dome Participant: {}", processId, ethereumAddress);
-        return domeParticipantRepository.findByEthereumAddress(ethereumAddress)
-                .switchIfEmpty(Mono.error(new UnauthorizedDomeParticipantException("Dome Participant not found")));
+    public Mono<Void> validateDomeParticipant(String processId, String dltAddress) {
+        log.info("ProcessID: {} - Validating Dome Participant: {}", processId, dltAddress);
+        return trustedAccessNodesListRepository.existsDltAddressByValue(Mono.just(dltAddress))
```

Comment:
check if methodParameter (data) exists in HashSet (.contains()) trustFrameworkConfig.getParticipantSet().contains(data); --> return true or false